src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/Blocks/configuration_form.html.twig

```diff
@@ -0,0 +1,64 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% block mailThemeConfigurationFormBlock %}
+  <div class="row justify-content-center">
+    <div class="col-xl-10">
+      {{ form_start(mailThemeConfigurationForm, {'action': path('admin_mail_theme_save_configuration')}) }}
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">settings</i> {{ 'Configuration'|trans }}
+        </h3>
+
+        <div class="card-block row">
+          <div class="card-text">
+
+            <div class="form-group row">
+              <label class="form-control-label">
+                {{ ps.label_with_help(
+                  ('Select your default mail theme'|trans),
+                  ('This won\'t regenerate mail templates, it only sets the default mail theme for future generation (when a language is installed for example).'|trans({}, 'Admin.MailTheme.Help'))
```

Comment:
I have a doubt regarding the domain _Admin.MailTheme.Help_, is it a module or a core feature? It will help me know where to localize this/these keys. ;-)